website/src/css/customTheme.css

```diff
@@ -292,7 +292,7 @@ header h2 {
   display: inline-flex;
   align-items: center;
   margin-bottom: 15px;
-  padding: 18px 18px;
```

Comment:
Why not just use `justify-content: center`?

Reply:
Yup, that fine, just changing the padding was the first thought that came to my mind. 
I too feel adding `justify-content: center` would make the css code more readable anyway.